multisig-wallet/src/main.sw

```diff
@@ -91,27 +101,40 @@ impl MultiSignatureWallet for Contract {
             Result::Ok(address) => address.value, _ => revert(42), 
         };
 
-        require(get::<bool>(signer1) && get::<bool>(signer2), Error::NotAnOwner);
         require(~b256::min() < signer1 && signer1 < signer2, Error::IncorrectSignerOrdering);
 
+        let signer1_weight = get::<u64>(signer1);
+        let signer2_weight = get::<u64>(signer2);
+
+        require(signer1_weight != 0 && signer2_weight != 0, Error::NotAnOwner);
```

Comment:
Would here be a good place to use the is_owner function defined below?  Or is the is_owner function more for external calls?